src/main/java/com/vmware/vipclient/i18n/base/cache/FormatCacheItem.java

```diff
@@ -4,28 +4,58 @@
  */
 package com.vmware.vipclient.i18n.base.cache;
 
-import java.util.HashMap;
-import java.util.Map;
+import com.vmware.vipclient.i18n.VIPCfg;
 
 public class FormatCacheItem implements CacheItem {
+
+	private String etag;
+	private long timestamp;
+	private Long maxAgeMillis = 86400000l;
```

Comment:
Maybe let's do code clean up in a separate PR because this affects MessageCacheItem too. If you both agree, @huihuiw01 , please create an issue to track this.